### DISCUSSION TITLE
Version 1.0.2 🎉

### DISCUSSION THREAD

**Discussion by AmarnathCJD** [AmarnathCJD/gogram#7] [Announcements]
**Full Changelog**: https://github.com/AmarnathCJD/gogram/compare/1.0...1.0.1

<hr /><em>This discussion was created from the release <a href='https://github.com/AmarnathCJD/gogram/releases/tag/1.0.2'>Version 1.0.2 🎉</a>.</em>

**Comment by AmarnathCJD** [comment]
@tamilvip007 gey

**Reply by IndrajeethY** [reply]
Nou
